src/hooks/useFirestoreConnect.ts

```diff
@@ -1,9 +1,11 @@
+// eslint-disable-next-line no-restricted-imports
 import {
   ReduxFirestoreQuerySetting,
-  useFirestoreConnect,
+  useFirestoreConnect as _useFirestoreConnect,
   isLoaded as _isLoaded,
   isEmpty as _isEmpty,
 } from "react-redux-firebase";
+
 import { ValidFirestoreKeys } from "types/Firestore";
 
 /**
```

Comment:
`Defined<T>` is already defined in `src/types/utility.ts` so doesn't need to exist here